packages/analytics-core/src/types/element-interactions.ts

```diff
@@ -50,6 +50,14 @@ export interface ElementInteractionsOptions {
    */
   pageUrlAllowlist?: (string | RegExp)[];
 
+  /**
+   * List of page URLs to exclude from auto tracking.
+   * When provided, tracking will be blocked on these URLs.
+   * Both full URLs and regex are supported.
+   * This is checked after pageUrlAllowlist.
```

Comment:
before/after isn't the right way to describe it maybe. `pageUrlExcludelist` takes precedence over `pageUrlAllowlist`